sunpy/lightcurve/tests/test_rhessi.py

```diff
@@ -55,21 +55,22 @@ def test_header(self, timerange_b):
     @pytest.mark.online
     def test_hsi_url(self):
         """Test creation with url"""
-        url = 'http://hesperia.gsfc.nasa.gov/hessidata/metadata/catalog/hsi_obssumm_20030302_146.fits'
+        url = 'http://soleil.i4ds.ch/hessidata/metadata/catalog/hsi_obssumm_20030302_146.fits'
         lc1 = sunpy.lightcurve.RHESSISummaryLightCurve.create(url)
         assert isinstance(lc1, sunpy.lightcurve.RHESSISummaryLightCurve)
 
     @pytest.mark.online
     def test_filename(self, timerange_a, timerange_b):
         """Compare data from two different time ranges to make
         sure they are not the same"""
-        lc1 = sunpy.lightcurve.RHESSISummaryLightCurve.create(timerange_a)
-        lc2 = sunpy.lightcurve.RHESSISummaryLightCurve.create(timerange_b)
-        assert not all(lc1.data[lc1.data.columns[0]] == lc2.data[lc2.data.columns[0]])
+        lc1 = np.array(sunpy.lightcurve.RHESSISummaryLightCurve.create(timerange_a).data)
+        lc2 = np.array(sunpy.lightcurve.RHESSISummaryLightCurve.create(timerange_b).data)
+        with pytest.raises(AssertionError):
+            np.testing.assert_allclose(lc1, lc2)
```

Comment:
it raises an `AssertionError` if they are not the same, which the `with` statement expects, so if it were not to raise an `AssertionError` the test would fail. Weird I know.